src/components/elements/shareCourses.tsx

```diff
@@ -59,20 +59,20 @@ const ShareCourses = ({courseName, courseUrl, courseNum}: Props) => {
         )}
       >
         <div className="absolute -top-3 left-3/4 z-0 h-10 w-10 rotate-45 bg-white" />
-        <ul className="list-unstyled bg-white children:border-b children:border-spirited-light last:children:border-transparent">
-          <li className="relative m-0 text-nowrap py-0">
+        <ul className="list-unstyled bg-white">
+          <li className="relative m-0 text-nowrap border-b border-spirited-light py-0 hocus:border-transparent">
             <a
               href={emailCopy}
-              className="block border-b-2 border-transparent px-6 py-5 font-normal text-black no-underline hocus:border-spirited-light hocus:text-black"
+              className="block border-b-[4px] border-transparent px-6 py-5 font-normal text-black no-underline hocus:border-spirited-light hocus:text-black"
               data-course-shared={courseNum}
             >
               Share via email
             </a>
           </li>
-          <li className="relative m-0 text-nowrap py-0">
+          <li className="relative m-0 text-nowrap border-b border-spirited-light py-0 hocus:border-transparent">
```

Comment:
hocus on the `<li>`? Do you mean to put these on the link below?

Reply:
The hocus is needed on the `<li>` to prevent the border lines from stacking with the nested `<a>` hocus state.

Reply:
Well it will only work for hover, not for focus. If you accept that caveat I don't mind this.

Reply:
You could accomplish the same effect by refactoring the markup and where the borders are applied (this is very roughly though out):

```
<li>
  <a className="group border-b-4 border-transparent hocus:border-cardinal-red">
    <span className="border-b border-cardinal-red group-hocus:border-0">
      {title}
    </span>
  </a>
</li>
```

Reply:
That sounds like a far better alternative. I'll get that in right now! Thank you, Mike!